Libraries/LibDraw/Triangle.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2018-2020, Shannon Booth <shannon.ml.booth@gmail.com>
```

Comment:
Unless you've been working on this class since 2018, this should read:

```suggestion
 * Copyright (c) 2020, Shannon Booth <shannon.ml.booth@gmail.com>
```

Reply:
Woops!